gcp-http-client/src/main/java/io/micronaut/gcp/http/client/GoogleAuthFilter.java

```diff
@@ -49,15 +52,15 @@
 @Requires(property = "gcp.http.client.auth.patterns")
 @Filter(patterns = "${gcp.http.client.auth.patterns:/**}")
 public class GoogleAuthFilter implements HttpClientFilter, AutoCloseable {
+
     private static final String METADATA_FLAVOR = "Metadata-Flavor";
     private static final String GOOGLE = "Google";
-    private static final String AUDIENCE = "/computeMetadata/v1/instance/service-accounts/default/identity?audience=";
+    private static final String IDENTITY_TOKEN_URI = "/computeMetadata/v1/instance/service-accounts/default/identity?audience=";
     private final HttpClient authClient;
+    private final ApplicationContext applicationContext;
 
-    /**
-     * Default constructor.
-     */
-    public GoogleAuthFilter() {
+    public GoogleAuthFilter(ApplicationContext applicationContext) {
```

Comment:
Please retain the original constructor to avoid the breaking binary change. You can check for the null in the logic that locates the audience instead.

Reply:
Can I inject the application context via `@Inject `then? And what do you mean with this sentence: 

> You can check for the null in the logic that locates the audience instead.

Reply:
I think I understood what you mend and did another commit.

Reply:
done